spec/bolt_ext/server_spec.rb

```diff
@@ -1,11 +1,10 @@
 # frozen_string_literal: true
 
-ENV['RACK_ENV'] = 'test'
-
 require 'bolt_spec/conn'
 require 'bolt_ext/server'
 require 'json'
 require 'rack/test'
+require 'spec_helper'
```

Comment:
This should probably be first.

Reply:
Oh, ha, I just alphabetized with abandon :sweat_smile: